recipes/openms-meta/meta.yaml

```diff
@@ -36,24 +42,34 @@ requirements:
     build:
       - {{ compiler('cxx') }}
       - cmake
-      - make
+      - ninja
       - autoconf
       - automake
-      - sysroot_linux-64 =2.17 # [linux]
-    host:
-      - llvm-openmp # [osx]
-      - zlib
       - libtool
+      - pkg-config
+      - make
+      - qt6-main
+      - sysroot_linux-64 =2.17  # [linux]
+    host:
+      - libgomp  # [linux]
+      - llvm-openmp  # [osx]
       - xerces-c
-      - boost-cpp
-      - eigen >=3.3.4
-      - hdf5 >=1.10
-      - bzip2
-      - qt-main
+      - libboost-devel
+      - libboost
+      - eigen >=3.4.0
+      - hdf5
       - libsvm
       - coinmp
-      - sysroot_linux-64 =2.17 # [linux]
       - yaml-cpp
+      - doxygen >=1.8.13
+      - sysroot_linux-64 =2.17  # [linux]
+      - zlib
+      - bzip2
+      - apache-arrow-proc =*=cpu # chooses the CPU, not CUDA version
+      - libparquet
+    run:
+      - libgomp  # [linux]
```

Comment:
I would be surprised if the OpenMP libs did not have run-exports. Try without run section first.